lib/ui/sign_in.dart

```diff
@@ -574,18 +592,85 @@ class SignInWidgetState extends AssistantState<SignInWidget> {
                             recognizer: TapGestureRecognizer()
                               ..onTap = () {
                                 launchUrl(url4);
-                              }
+                              }),
+                      ]))),
+                      Container(
+                        padding: const EdgeInsets.only(top: 10),
+                        child: Column(
+                          children: [
+                            Row(
+                              children: [
+                                Expanded(
+                                    child: Container(
+                                  padding: const EdgeInsets.only(
+                                      left: 35, right: 35),
+                                  child: SignInButton(
+                                    Buttons.Google,
+                                    onPressed: () async {
+                                      // To get oauth token
+                                      bool success = await UserAuthService()
+                                          .signInGoogleEmail();
+
+                                      if (success) {
+                                        await CacheService.updateMail("", "");
+                                        Navigator.pushNamed(context, '/main');
+                                      } else {
+                                        showLoginErrorDialog();
+                                        context.loaderOverlay.hide();
+                                      }
+                                    },
+                                    text: 'Sign In with Google',
+                                  ),
+                                ))
+                              ],
+                            ),
+                            Container(
+                              padding: const EdgeInsets.only(top: 10),
+                            ),
+                            Row(
+                              children: [
```

Comment:
apple and MS sign in require developer account to configure. you should remove it unless its work in progress.

Reply:
@mdconatser Are the other sign in options being actively worked on?

Reply:
i don't believe anyone is working on the other ones yet

Reply:
removed apple/ms